### ISSUE TITLE
[a11y]: No focus indicator for toggle/small toggle on Mac

### ISSUE THREAD

**Issue by RachelGHogan** [carbon-design-system/carbon#17626]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.67.0

### Automated testing tool and ruleset

Chrome Devtools High Contrast Mode

### Description

When on a Mac with chrome devtools forced-colors active, there is no focus indicator when focus is on a toggle or small toggle.

![image](https://github.com/user-attachments/assets/afd728a1-0ed9-4398-a14f-82925b580657)

### Reproduction/example

https://stackblitz.com/edit/github-5skpvg?file=src%2FApp.jsx

### Steps to reproduce

1. Using a Mac, open the stackblitz by clicking the link above in the reproduction section
2. Open chrome devtools
3. Set "Emulate CSS media feature forced-colors" to active
4. Focus on one of the toggles to see that there is no focus indicator

### Suggested Severity

None

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Thanks for the report! This might be either a bug with macos that we could work around, or just a plain defect on our end. HCM is typically tested in windows and not as often in mac, which could be why this issue exists.
